byterun/caml/shared_heap.h

```diff
@@ -22,8 +22,10 @@ intnat caml_sweep(struct caml_heap_state*, intnat);
 /* must be called during STW */
 void caml_cycle_heap_stw(void);
 
-/* must be called on each domain 
+/* must be called on each domain
    (after caml_cycle_heap_stw) */
 void caml_cycle_heap(struct caml_heap_state*);
 
+int is_garbage (value);
```

Comment:
The value `global.GARBAGE` is local to `shared_heap.c`. `is_garbage` is now only defined under DEBUG mode. Hopefully, this makes it more clear where `is_garbage` should be used.